ITR/data/data_warehouse.py

```diff
@@ -2,16 +2,21 @@
 import logging
 import pandas as pd
 import numpy as np
+
 from abc import ABC
 from typing import List, Type
 from pydantic import ValidationError
 
-from ITR.interfaces import IEmissionRealization, IEIRealization, ICompanyAggregates, ICompanyEIProjection
+import ITR
```

Comment:
same comment about `import ITR`

Reply:
Yes, and I give back the same answer ;-). I'll leave the first case of the argument open to discuss and close this as resolved.  If we decide to make a change, we'll make it everywhere...